feature1/feature1.txt

```diff
@@ -0,0 +1,2 @@
+This file was added in feature1 branch.
```

Comment:
I t is good to see that you have added this line of code.

Reply:
Thanks for approval and your time